mimoto-issuers-config.json

```diff
@@ -228,127 +228,33 @@
           },
-          "title": "National Identity Department (Certify)",
-          "description": "Download MOSIP National / Foundational Identity Credential",
+          "title": "Mock Identity",
+          "description": "Download Mock Identity Credential",
           "language": "en"
-        },
-        {
-          "name": "دائرة الهوية الوطنية",
-          "logo": {
-            "url": "https://${mosip.api.public.host}/inji/mosip-logo.png",
-            "alt_text": "شعار موسيب"
-          },
-          "title": "دائرة الهوية الوطنية",
-          "description": "قم بتنزيل بيانات اعتماد الهوية الوطنية / التأسيسية MOSIP",
-          "language": "ar"
-        },
-        {
-          "name": "राष्ट्रीय पहचान विभाग",
-          "logo": {
-            "url": "https://${mosip.api.public.host}/inji/mosip-logo.png",
-            "alt_text": "मोसिप लोगो"
-          },
-          "title": "राष्ट्रीय पहचान विभाग",
-          "description": "MOSIP नेशनल/फाउंडेशनल आइडेंटिटी क्रेडेंशियल डाउनलोड करेंं",
-          "language": "hi"
-        },
-        {
-          "name": "ರಾಷ್ಟ್ರೀಯ ಗುರುತಿನ ಇಲಾಖೆ",
-          "logo": {
-            "url": "https://${mosip.api.public.host}/inji/mosip-logo.png",
-            "alt_text": "mosip ಲೋಗೋ"
-          },
-          "title": "ರಾಷ್ಟ್ರೀಯ ಗುರುತಿನ ಇಲಾಖೆ",
-          "description": "MOSIP ರಾಷ್ಟ್ರೀಯ / ಫೌಂಡೇಶನಲ್ ಐಡೆಂಟಿಟಿ ರುಜುವಾತು ಡೌನ್\u200Cಲೋಡ್ ಮಾಡಿ",
-          "language": "kn"
-        },
-        {
-          "name": "தேசிய அடையாளத் துறை",
-          "logo": {
-            "url": "https://${mosip.api.public.host}/inji/mosip-logo.png",
-            "alt_text": "mosip லோகோ"
-          },
-          "title": "தேசிய அடையாளத் துறை",
-          "description": "MOSIP தேசிய / அடிப்படை அடையாளச் சான்றிதழைப் பதிவிறக்கவும்",
-          "language": "ta"
-        },
-        {
-          "name": "National Identity Department",
-          "logo": {
-            "url": "https://${mosip.api.public.host}/inji/mosip-logo.png",
-            "alt_text": "logo ng mosip"
-          },
-          "title": "National Identity Department",
-          "description": "I-download ang MOSIP National / Foundational Identity Credential",
-          "language": "fil"
         }
       ],
       "protocol": "OpenId4VCI",
-      "client_id": "${mimoto.oidc.partner.clientid}",
-      "client_alias": "mpartner-default-mimotooidc",
-      "scopes_supported": [
-        "mosip_identity_vc_ldp"
-      ],
-      "additional_headers": {
-        "Accept": "application/json"
-      },
-      ".well-known": "https://${mosip.injicertify.mosipid.host}/v1/certify/issuance/.well-known/openid-credential-issuer?version=vd11",
+      "client_id": "${mimoto.mock.oidc.clientid}",
+      "client_alias": "mpartner-mock-testing",
+      "scopes_supported": ["mosip_identity_vc_ldp"],
+      "additional_headers": { "Accept": "application/json" },
+      ".well-known": "https://${mosip.esignet.mock.host}/.well-known/openid-credential-issuer?version=v11",
```

Comment:
this should be `mock.identity.esignet.host`

Reply:
updated